config/migrations/20240916165708-remove-old-concept.sparql

```diff
@@ -0,0 +1,13 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+DELETE {
```

Comment:
I don't think it's enough,  but we need to migrate the old values as well, in this case we just delete the concept; if it's not a requirement to migrate the old values, then it's ok (tbc with @nvdk) . Also the `ordering` property still has to be migrated (unless we can get rid of it and start from scratch)

Reply:
For now we could keep the old concept and just add the `valid` property. Then we could tackle this issue in https://binnenland.atlassian.net/jira/software/c/projects/GN/boards/303?issueParent=74128&selectedIssue=GN-4948, or make a new ticket? @nbittich @nvdk